codegenerator/cli/npm/envio/src/LazyLoader.res

```diff
@@ -99,37 +99,37 @@ let rec loadNext = async (am: asyncMap<'a>, k: int) => {
     switch await awaitTaskPromiseAndLoadNextWithTimeout() {
     | _ => Promise.resolve()
     | exception err =>
-      Logging.error({
-        "err": err,
-        "msg": `EE1100: Top level promise timeout reached. Please review other errors or warnings in the code. This function will retry in ${(am._retryDelayMillis / 1000)
-            ->Belt.Int.toString} seconds. It is highly likely that your indexer isn't syncing on one or more chains currently. Also take a look at the "suggestedFix" in the metadata of this command`,
-        "metadata": am.metadata,
-      })
-      await Time.resolvePromiseAfterDelay(~delayMilliseconds=am._retryDelayMillis)
+      switch am.onError {
+      | None => ()
+      | Some(onError) => onError(am, ~exn=err)
+      }
```

Comment:
A thought I had with this side effect logging, is that we may want to be able clean up/cancel this `onError` (or even the actual task) from where the lazy loader is called. In case you want to implemented a higher level timeout etc. Then you don't have to worry about stale logs and retries after you've already handled it etc.

Reply:
I scoped out all the timeout handling changes for now